.ci/install_cri_containerd.sh

```diff
@@ -0,0 +1,56 @@
+#!/bin/bash
+#
+# Copyright (c) 2018 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+script_dir="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
+cri_repository="github.com/containerd/cri"
+
+die() {
+	echo >&2 -e "ERROR: $*"
+	exit 1
+}
+
+info() {
+	echo -e "INFO: $*"
+}
+
+source "${script_dir}/lib.sh"
+cri_containerd_version=$(get_version "externals.cri-containerd.version")
+containerd_version=$(get_version "externals.cri-containerd.meta.containerd-version")
+
+
+case "$(arch)" in
+	"aarch64")
+		ARCH=arm64
+		;;
+
+	"x86_64")
+		ARCH=amd64
+		;;
+	"*")
+		die "Arch $(arch) not supported"
+		;;
+esac
+
+
+go get github.com/containerd/cri
+pushd "${GOPATH}/src/${cri_repository}" >> /dev/null
+git fetch
+git checkout "${cri_containerd_version}"
+make install.tools
+make install.deps
+popd  >> /dev/null
+
+curl -OL \
+https://storage.googleapis.com/cri-containerd-release/cri-containerd-${containerd_version}.linux-${ARCH}.tar.gz
+sudo tar -C / -xzf "cri-containerd-${containerd_version}.linux-${ARCH}.tar.gz"
```

Comment:
Can't you use `.ci/install_go.sh` here?

Reply:
why in `/` and not in a temporal directory? I wouldn't like to run this script in my workstation  :worried:

Reply:
@devimc  @devimc  I moved to do an installation from sources that probably make you more confident instead of untar all in root , let me know what version you prefer

Reply:
I'll prefer to install from tarball instead of building from sources, but only if this saves us time. If not, I think either way is ok.